android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1687,4 +1687,14 @@ public static double parseDouble(String s) throws ParseException {
 
         return num.doubleValue();
     }
+
+    /**
+     * Gets truncated address from a full contract address.
+     * @param address full contract address
+     * @return truncated address
+     */
+    public static String getTruncatedAddress(String address) {
+        int prefixLength = address.startsWith("0x") ? 6 : 4;
+        return (address.substring(0, prefixLength) + "…" + address.substring(address.length() - 4));
+    }
```

Comment:
nit: let's add these kind of methods into specific `Utils` class going forward, like `AddressUtils`